webui/react/src/components/Table/Table.tsx

```diff
@@ -152,6 +153,28 @@ export const taskNameRenderer: TaskRenderer = (id, record) => (
   </div>
 );
 
+export const taskWorkspaceRenderer = (
+  record: CommandTask,
+  workspaces: Workspace[],
+): React.ReactNode => {
+  const workspace = workspaces.find((u) => u.id === record.workspaceId);
+  const workspaceId = record.workspaceId;
+  const isUncategorized = workspaceId === 1;
+
+  return (
+    <Tooltip placement="top" title={workspace?.name}>
+      <div className={`${css.centerVertically} ${css.centerHorizontally}`}>
+        <Link
+          path={
+            isUncategorized ? paths.projectDetails(workspaceId) : paths.workspaceDetails(workspaceId)
```

Comment:
nit: I realized that there's not any way to see the tasks in the Uncategorized workspace if we link directly to the project. I still don't think we should show the Workspace page, but this could be confusing.